frontend/src/pages/ActivityReport/Pages/components/Objective.js

```diff
@@ -406,7 +441,41 @@ export default function Objective({
         />
       </FormFieldThatIsSometimesReadOnly>
       {
-        citationOptions.length > 0 && (
+        isMonitoringGoal && citationWarnings.length > 0 && (
+          <Alert type="warning" className="margin-bottom-2">
+            <span>
+              <span className="margin-top-0">
+                {citationWarnings.length > 1
+                  ? 'These grants do not have any of the citations selected:'
+                  : 'This grant does not have any of the citations selected:'}
+                <ul className="margin-top-2">
+                  {citationWarnings.map((grant) => (
+                    <li key={grant}>{grant}</li>
+                  ))}
+                </ul>
+              </span>
+              <span className="margin-top-2 margin-bottom-0">
+                To avoid errors when submitting the report, you can either:
+                <ul className="margin-top-2 margin-bottom-0">
+                  <li>
+                    Add a citation for this grant under an objective for the monitoring goal
+                  </li>
+                  <li>
+                    Remove the grant from the
+                    {' '}
+                    <Link to={`/activity-reports/${reportId}/activity-summary`}>Activity summary</Link>
+                  </li>
+                  <li>
+                    Add another goal to the report
+                  </li>
+                </ul>
+              </span>
+            </span>
+          </Alert>
```

Comment:
are these alerts identical except for the type? Might be a good idea to turn these into a component

Reply:
I think there are enough differences that the component will be simply and alert.